content/03.methods.md

```diff
@@ -0,0 +1,36 @@
+## Methods
+
+### Mutation data download and preprocessing
+
+To generate binary mutated/non-mutated gene labels for our machine learning model, we used mutation calls for TCGA samples from MC3 [@doi:10.1016/j.cels.2018.03.002] and copy number threshold calls from GISTIC2.0 [@doi:10.1186/gb-2011-12-4-r41].
+MC3 mutation calls were downloaded from the Genome Data Commons (GDC) of the National Cancer Institute, at [`https://gdc.cancer.gov/about-data/publications/pancanatlas`](https://gdc.cancer.gov/about-data/publications/pancanatlas).
+Copy number threshold calls are from an older version of PanCanAtlas, and are available here: [`https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122`](https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122).
+We removed hypermutated samples (defined as five or more standard deviations above the mean non-silent somatic mutation count) from our dataset, to avoid potential false positives.
```

Comment:
```suggestion
We removed hypermutated samples (defined as five or more standard deviations above the mean non-silent somatic mutation count) from our dataset to reduce the number of false positives (i.e., non-driver mutations).
```
Don't love this edit but figured we needed to somehow note why we avoided these more explicitly. I'm not sure if non-driver is the best term to use though - it feels loaded.

Reply:
I'll go with this for now, but I'll keep thinking about it. "Non-driver" seems to capture the right idea.